ucm2/Motorola/ocean/HiFi.conf

```diff
@@ -0,0 +1,123 @@
+# Use case configuration for Motorola Moto G7 Power
+# Speaker connected to external code (TFA9874)
+
+SectionVerb {
+	EnableSequence [
+		cset "name='PRI_MI2S_RX Audio Mixer MultiMedia1' 1"
+		cset "name='QUIN_MI2S_RX Audio Mixer MultiMedia3' 1"
+		cset "name='MultiMedia2 Mixer TERT_MI2S_TX' 1"
+	]
+	DisableSequence [
+		cset "name='PRI_MI2S_RX Audio Mixer MultiMedia1' 0"
+		cset "name='QUIN_MI2S_RX Audio Mixer MultiMedia3' 0"
+		cset "name='MultiMedia2 Mixer TERT_MI2S_TX' 0"
+	]
+
+	Value {
+		TQ "HiFi"
+	}
+}
+
+SectionDevice."Speaker" {
+	Comment "Speaker playback"
+
+	ConflictingDevice [
+		"Earpiece"
+	]
+
+	Value {
+		PlaybackPCM "hw:${CardId},2"
+		PlaybackPriority 200
+		PlaybackChannels 1
+	}
+}
+
+#Speaker is also the Earpiece
```

Comment:
If speaker and earpiece is same than you should keep Speaker, and remove Earpiece i don't see any advantage of keep both of them.